rbac/management/relation_replicator/sync_replicator.py

```diff
@@ -0,0 +1,81 @@
+#
+# Copyright 2024 Red Hat, Inc.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+"""RelationReplicator which writes to the outbox table."""
+
+import json
+import logging
+
+
+import grpc
+from django.conf import settings
+from google.rpc import error_details_pb2
+from grpc_status import rpc_status
+from kessel.relations.v1beta1 import relation_tuples_pb2
+from kessel.relations.v1beta1 import relation_tuples_pb2_grpc
+from management.relation_replicator.relation_replicator import RelationReplicator, ReplicationEvent
+
+
+logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
+
+
+class SyncReplicator(RelationReplicator):
```

Comment:
just a nitpick:
I’m thinking about whether the name is descriptive enough, but all I have so far is:
`GrpcSyncReplicator`

But maybe it’s just me :)